SYCL/Basic/half_builtins.cpp

```diff
@@ -0,0 +1,227 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out

```

Comment:
There are 3 ways to fix the pre-commit:
1. Remove execution on CPU
2. Mark as expected to fail on CPU or OpenCL CPU -> `XFAIL: cpu` or `XFAIL: cpu && opencl`
3. Add "UNSUPPORTED: cpu".

I prefer 3 as it saves the time to run the test.